cadctl/cmd/cluster-missing/cluster-missing.go

```diff
@@ -85,6 +85,13 @@ func run(cmd *cobra.Command, args []string) error {
 
 	fmt.Printf("ClusterExternalID is: %s\n", externalClusterID)
 
+	cloudProvider, err := ocmClient.GetCloudProvider(externalClusterID)
+	if err != nil {
+		return fmt.Errorf("GetCloudProvider failed on: %w", err)
```

Comment:
**todo:** Remove duplicate error text. You can just return err here, as the error message is formatted that way already in the `GetCloudProvider` function